Userland/Applications/PixelPaint/Tools/Tool.h

```diff
@@ -64,6 +64,7 @@ class Tool {
     virtual bool on_keydown(GUI::KeyEvent const&);
     virtual void on_keyup(GUI::KeyEvent&) { }
     virtual void on_tool_activation() { }
+    virtual void on_tool_exit() { }
```

Comment:
`on_tool_deactivation` would be more clearly related to `on_tool_activation`

Reply:
Thanks for the suggestion. I've changed this in latest push.